### PR TITLE
CLOUDP-94311: Include generated docs in release tasks an notifications

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-94311

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Genertae docs on release and include on our internal announcement a link to it so docs can easily grave them

### CODE REVIEW

File: build/package/send-slack-notification.sh [hunk 1]
```diff
@@ -28,8 +28,15 @@ curl --header "Api-User:${evergreen_user:?}" \
        {
        	"title": "v'"${VERSION}"'",
        	"title_link": "https://github.com/mongodb/mongocli/releases/tag/v'"${VERSION}"'",
-       	"author_name": "MongoDB CLI",
+       	"author_name": "Release Page",
        	"fallback": "new release",
 	      "author_icon": "https://camo.githubusercontent.com/0e10b56a03b056a6e5fcf82a3cb3603188549a02c234b8e5426aaa11853e3069/68747470733a2f2f7261772e6769746875622e636f6d2f6d6f6e676f64622f6d6f6e676f636c692f6d61737465722f6d6f6e676f636c692e706e67"
+       },
+       {
+       	"title": "command.tgz",
```
Comment (andreaangiolillo) [inline build/package/send-slack-notification.sh:36]: [q] Is this title correct? 👀

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: build/package/send-slack-notification.sh (hunk 1 above)
Comment (gssbzn) [reply build/package/send-slack-notification.sh:36]: that's the name of the file we generate

**Review by colm-quinn — APPROVED** [review]
LGTM

**Pull request merged**